components/me.js

```diff
@@ -10,6 +10,10 @@ export const MeContext = React.createContext({
 export function MeProvider ({ me, children }) {
   const { data } = useQuery(ME, SSR ? {} : { pollInterval: 1000, nextFetchPolicy: 'cache-and-network' })
 
+  const futureMe = data?.me || me
+  // weblnSats is initially undefined on page load for some reason
```

Comment:
is the WebLN provider initialized yet?

Reply:
It should be, it's above the `MeProvider` in _\_app.js_:

```js
<WebLNProvider>
  <ApolloProvider client={client}>
    <MeProvider me={me}>
```

I think it's related to how Apollo works since this is a [local-only field](https://www.apollographql.com/docs/react/local-state/managing-state-with-field-policies/#storing-local-state-in-reactive-variables) resolved with a local resolver in _apollo.js_:

```javascript
resolvers: {
  User: {
    weblnSats: (user, args, { cache }) => {
      const balance = WebLNContextRef.current?.balance
      return balance
    }
  }
},
```

But in `WebLNProvider`, I am setting `balance` to 0 by default :thinking: Need to look into this further.